ambry-tools/src/main/java/com.github.ambry/tools/perf/rest/NettyPerfClient.java

```diff
@@ -188,10 +205,12 @@ protected ClientArgs(String args[]) {
       this.concurrency = options.valueOf(concurrency);
       this.postBlobTotalSize = options.valueOf(postBlobTotalSize);
       this.postBlobChunkSize = options.valueOf(postBlobChunkSize);
-      this.sslPropsFilePath = options.valueOf(sslPropsFilePath);
       this.targetAccountName = options.valueOf(targetAccountName);
       this.targetContainerName = options.valueOf(targetContainerName);
       this.serviceId = options.valueOf(serviceId);
+      this.testTime = options.valueOf(testTime);
```

Comment:
is this 0 if not set?

Reply:
It will be `null`. I added a condition to not schedule shutdown if `testTime` is not set.